### PR TITLE
[CARMAN-213] Move Page Title To Package

### PR DESCRIPTION
**Pull request by DAArbelaez**
## Jira ticket
[CARMAN-213](https://danchez.atlassian.net/browse/CARMAN-213)

### Description
Move CMPageTitle widget to package.

### Evidence
WhatsApp Image 2024-08-08 at 10 31 57 AM

### Depends on
Link to pull requests that are needed for this PR. If this PR depends on other PR, please add Don’t merge label.

### CODE REVIEW

File: lib/src/components/page_title/page_title.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            lightTitle,
+            maxLines: 1,
+            overflow: TextOverflow.ellipsis,
+            style: kSubtitleTextStyle.copyWith(fontWeight: FontWeight.w300),
+          ),
+          Text(
+            boldTitle,
+            style: kTitleTextStyle.copyWith(fontWeight: FontWeight.w600),
+          ),
+          if (showBottomDivider) kPrimaryDivider
```
Comment (dan-chez) [inline lib/src/components/page_title/page_title.dart:50]: Can we use Visibility widget please. Just to keep the same pattern for these cases

File: lib/src/components/page_title/page_title.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+///
+/// Example usage:
+/// ```dart
+/// CMPageTitle(
+///   lightTitle: "Welcome to",
+///   boldTitle: "Flutter",
+///   showBottomDivider: true,
+/// );
+/// ```
+class CMPageTitle extends StatelessWidget {
```
Comment (dan-chez) [inline lib/src/components/page_title/page_title.dart:23]: Should we keep calling this CMPageTitle or just PageTitle?

Comment (dan-chez) [reply lib/src/components/page_title/page_title.dart:23]: Class name is page_title...

File: lib/src/constants/text_constants.dart [hunk 3]
```diff
@@ -15,6 +15,17 @@ const kCMOutLinedButtonTextStyle = TextStyle(
   color: kAmaranthPrimary,
 );
 
+const kSubtitleTextStyle = TextStyle(
```
Comment (dan-chez) [inline lib/src/constants/text_constants.dart:18]: Are these styles for buttons?

**Review by dan-chez — CHANGES_REQUESTED** [review]

File: lib/src/components/page_title/page_title.dart (hunk 2 above)
Comment (DAArbelaez) [reply lib/src/components/page_title/page_title.dart:23]: cm... is fine.

**Review by dan-chez — APPROVED** [review]

**Pull request merged**